rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -148,21 +151,24 @@ without guessing. The format of the value may depend on the `ppt_scheme` attribu
 
 {align="left"}
 
-| Attribute      | Required? | Value                       |
-|----------------|-----------|-----------------------------|
-| ppt_scheme     | Y         | mqtt                        |
-| ppt_serializer | N         | json, msgpack, cbor, native |
-| ppt_cipher     | N         | -                           |
-| ppt_keyid      | N         | -                           |
+| Attribute      | Required? | Value  |
+|----------------|-----------|--------|
+| ppt_scheme     | Y         | mqtt   |
+| ppt_serializer | N         | native |
```

Comment:
mqtt in CB support all of json, msgpack, cbor and native. pls remove the change to ppt_serializer